packages/avatar-bundle/src/AvatarBundle.js

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { cx, css } from "emotion";
+import { polyfill } from "react-lifecycles-compat";
```

Comment:
I don't think you need this `polyfill` anymore? It doesn't look like you are using any lifecycles.